charts/linkerd-crds/templates/policy/httproute.yaml

```diff
@@ -16,6 +16,7 @@ spec:
     listKind: HTTPRouteList
     plural: httproutes
     singular: httproute
+    shortNames: [route, httprt, rt]
```

Comment:
Hmm, I'm not actually sure if it's a good idea to add these shortnames for the `policy.linkerd.io.HTTPRoute` CRD, as the intention is to eventually replace it with the upstream version of HTTPRoute from the Gateway API. The upstream resource type does not have these shortnames: https://github.com/kubernetes-sigs/gateway-api/blob/e097dff9f32a3edd0e9465158fa7b48028e3dc86/config/crd/standard/gateway.networking.k8s.io_httproutes.yaml#L12-L18

This means that when we replace the vendored Linkerd version of this CRD with the upstream definition, the shortnames will disappear.

Also, I'm unsure if we really want to refer to HTTPRoute as "route" or "rt" when Linkerd will likely be adding support for the GRPCRoute and TCPRoute CRDs in the future as they are standardized by the gateway API...

Reply:
Thank you @hawkw! Sounds good to me! I'll remove all of the short names for the HTTPRoute type since it will be going away ultimately.

Reply:
@javaducky @hawkw please see https://github.com/linkerd/linkerd2/pull/10030#discussion_r1066914589